mysql/resource_rds_config.go

```diff
@@ -0,0 +1,182 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceRDSConfig() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: CreateRDSConfig,
+		UpdateContext: UpdateRDSConfig,
+		ReadContext:   ReadRDSConfig,
+		DeleteContext: DeleteRDSConfig,
+		Importer: &schema.ResourceImporter{
+			StateContext: ImportRDSConfig,
```

Comment:
Is there any reason to use a custom Import function if you could pass it like this?
```
       StateContext: schema.ImportStatePassthroughContext,
```

That way, reading is done for you automatically and it's less code.

Reply:
Oh, I didn't know that, thanks. Will change.